lib/libc/stdio/lib_sscanf.c

```diff
@@ -70,8 +70,11 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
+#define CONFIG_DYNAMIC_ALLOC_ENABLE
```

Comment:
Let's make it in Kconfig. If you want to define this in here, please remove `CONFIG_`. But I think this is difficult to find and to control.